PROTOCOL.md

```diff
@@ -710,25 +807,39 @@ Writers should reject any transaction that contains data where the expression `x
 
 ## CHECK Constraints
 
+Enablement:
+- If the table is on a Writer Version up to 6, CHECK Constraints is enabled when the table's metadata contains required keys.
```

Comment:
Same: I would suggest saying that it's also enabled. And what is the minimum writer version?